pkg/local-gadget-manager/local-gadget-manager.go

```diff
@@ -302,6 +304,25 @@ func NewManager() (*LocalGadgetManager, error) {
 		return nil, err
 	}
 
+	// Ensure /sys/fs/bpf is of type bpf. It is necessary to be able to pin eBPF
+	// maps. TODO: Remove the need of using pinning, see issues #619 and #620.
+	cmd := exec.Command("/bin/sh", "-c", "stat -f --format=%T /sys/fs/bpf")
```

Comment:
Could you use https://pkg.go.dev/golang.org/x/sys/unix#Statfs?

BPF_FS_MAGIC = 0xcafe4a11 according to manual page statfs.2.

Reply:
Yeah, I should have expected that they exist, sorry.

I update the PR to use `unix.Mount` and `unix.Statfs`.